packages/driver/src/cy/navigation.ts

```diff
@@ -25,6 +26,11 @@ export const historyNavigationTriggeredHashChange = (state): boolean => {
   }
 
   const urls = state('urls')
+
+  if (_.isEmpty(urls)) {
```

Comment:
Let's explicitly check for a urlPosition value here too before continuing on. Your second spec asserts that it doesn't throw without a value, but the logic that follows isn't very meaningful without one.